key_handling.h

```diff
@@ -57,6 +63,33 @@ void register_keys(void);
  */
 void check_key_status(void);
 
+/*!
+ * Set backend public key.
+ *
+ * @param key The key in base64 string format, '\0' terminated
+ * @return ESP_OK
```

Comment:
the function header comment does not fit to the one, in `key handling.c`. Please use the form of the ..c file.
the double @return statement can be misleading

Reply:
Done. (And Calvin, don't worry, some other `@return` was meant :rofl:)

Reply:
This might also be my code. It would be nice if you can change that. We usually only do function fheader commenst in the `..h` files. So feel free to cut the comments from the `..c` files and paste them in the `..h` files

Reply:
The comments in the `..c` files are only on static functions that are not represented in the header, i'm not sure if we should delete them.